src/quo2/core_spec.cljs

```diff
@@ -40,4 +40,5 @@
     [quo2.components.settings.settings-list.component-spec]
     [quo2.components.settings.category.component-spec]
     [quo2.components.share.share-qr-code.component-spec]
-    [quo2.components.tags.--tests--.status-tags-component-spec]))
+    [quo2.components.tags.--tests--.status-tags-component-spec]
+    [quo2.components.numbered-keyboard.keyboard-key.component-spec]))
```

Comment:
actually same here, please keep this alpha sorted 👍 - not all files are alpha sorted but if you can see the pattern is mostly there then it's best to keep it that way and in some cases go one step further and help clean up the ones that are not :)

Reply:
Done